core/spec/models/spree/option_type_spec.rb

```
@@ -1,12 +1,15 @@
 require 'spec_helper'
 
-describe Spree::OptionType, :type => :model do
-  context "touching" do
-    it "should touch a product" do
+describe Spree::OptionType, type: :model do
+  context 'touching' do
+    it 'should touch a product' do
       product_option_type = create(:product_option_type)
       option_type = product_option_type.option_type
       product = product_option_type.product
-      product.update_column(:updated_at, 1.day.ago)
+      product.update_attributes(
```

Can you use `update_attributes!` here so it'll raise an error if the object is invalid?